src/App.css

```diff
@@ -0,0 +1,20 @@
+#the-circle-of-sounds {
+  height: 50vw;
```

Comment:
you are setting your height according to the view width. is this what you are expecting?

Reply:
If he wants to maintain a circle and have that circle size be dependent on vw then this works.